global/scripts/processingmodecheck.shlib

```diff
@@ -22,33 +22,48 @@ check_mode_compliance()
     # --- Initial information
 
     if [ "${ProcessingMode}" = "LegacyStyleData" ]; then
-      log_Msg "LegacyStyleData mode requested."
-      log_Msg "NOTICE: You are using a mode that enables processing of acquisitions that do not"
-      log_Msg "        conform to the HCP specification as described in Glasser et al. (2013)!"
-      log_Msg "        Be aware that if the HCP requirements are not met, the level of data quality"
-      log_Msg "        can not be guaranteed and the Glasser et al. (2013) paper should not be used"
-      log_Msg "        in support of this workflow. A manuscript with comprehensive evaluation for"
-      log_Msg "        the LegacyStyleData processing mode is in active preparation and should be"
-      log_Msg "        appropriately cited when published."
-      log_Msg "        "
+        log_Msg "LegacyStyleData mode requested.
+NOTICE: You are using a mode that enables processing of acquisitions that do not
+        conform to the HCP specification as described in Glasser et al. (2013)!
+        Be aware that if the HCP requirements are not met, the level of data
+        quality can not be guaranteed and the Glasser et al. (2013) paper should
+        not be used in support of this workflow. A manuscript with comprehensive
+        evaluation for the LegacyStyleData processing mode is in active
+        preparation and should be appropriately cited when published.
```

Comment:
While we are tweaking the message, should we remove the last sentence about a manuscript in "active preparation", which doesn't seem forthcoming any time soon...